GraphQL-TSC.md

```diff
@@ -37,7 +37,7 @@ Developers who are covered under a signed spec membership agreement are able to
 
 ## TSC meetings
 
-The GraphQL TSC will meet monthly, at the beginning of the [GraphQL Working Group meeting](https://github.com/graphql/graphql-wg). Our goal is to meet regularly to address any agenda items quickly and openly. By combining the TSC meeting with the open attendance Working Group meetings, we are ensuring that the broader community has visibility into the operations of the TSC, and vice versa.
+The GraphQL TSC will meet monthly, at the beginning of the first [GraphQL Working Group meeting](https://github.com/graphql/graphql-wg) each month. Our goal is to meet regularly to address any agenda items quickly and openly. By combining the TSC meeting with the open attendance Working Group meetings, we are ensuring that the broader community has visibility into the operations of the TSC, and vice versa.
```

Comment:
This is new, but I think makes sense.

An alt might be to allow TSC agenda at any meeting, but perhaps that's too difficult to keep track of

Reply:
In practice most TSC specific agenda/votes happen async anyhow

Reply:
Due to the tendency to add agenda items late, I think making it so any meeting could become a TSC meeting with only a few minutes notice is undesirable. If we were to go that route I’d like to see some kind of notice period on TSC topics (which with meetings being weekly should be less onerous than it is currently). I prefer the “first meeting” approach; though TBH we’ve had so few TSC topics in meetings it might be worth just making them ad-hoc.